Coalition News (#5836)
endless-sky/endless-sky: data/coalition/coalition news.txt

news "heliarch agent"
	location
		government "Coalition" "Heliarch"
	name
		word
			"Heliarch agent"
	message
		word
			`"Protect and serve."`
			`"If any assistance you need, at your disposal I am."`
			`"If any trouble you find, report it to us you must."`
			`"Saryd, Kimek, Arach, human, no matter what species, to protect all of our citizens, committed we are."`
			`"Enjoy my post well enough, I do, but still to get a break during the Coalition games to watch them, I am."`
			`"Part of a Punisher crew, I am. Land, refuel, rest, and back to working on that magnificent ship, I am!"`
			`"Offered the position of elder, I actually have been. Only, not interested in that governing activity, I am."`
			`"Apologies, but make haste I must. Gone missing some garrison guns have, and back to cleaning Stalker engines I will be, if find them I don't!"`
			`"Difficult sometimes it is, to be a mediator. If careful with my words I am not, a Saryd might think an Arach talking about longcows has insulted their mother."`

> 			"An agent is trying to calm down a small crowd outside a local bar. After some minutes, the more agitated drunkards are convinced to accompany the agent to a nearby Heliarch office."

Reviewer: To the best of my understanding, these messages will appear on the three rings. Considering that, it feels weird to me that there would be bars full of unruly drunkards in places dedicated almost solely to governance and research. Perhaps have that message appear on normal Coalition planets only, and not the three rings.

Author: Ringworlds are colossal structures, over a trillion Quarg can live in a complete one. All of the Coalition, 280 billion people, cannot hope to fill that, much less three (admittedly two being incomplete, but still), so the way I see it, while the Heliarchs are most prominent in the rings, and their primary functions are to serve as hubs for the Heliarch government, military and research teams, there is no reason to discourage civilians from settling there or having small businesses (which would also serve as amenities for the troops) there. Granted, many areas are restricted to non-Heliarchs, but I don't think they would stop a bar from being established near where civilian captains would go to either conduct business or relax.

Reviewer: That makes sense.

news "coalition merchant"
	location
		government "Coalition" "Heliarch"
	name
		word
			"Saryd captain"
			"Kimek captain"
			"Arach captain"
			"Saryd pilot"
			"Kimek pilot"
			"Arach pilot"
	message
		word
			`"Enough money for my first ship, I didn't have, so split the cost of a Runabout, me and an online friend did. Picked another ship, we should have, because too difficult flying alone to their planet, it was."`
			`"Enough money for my first ship, I didn't have, so a loan from house Plumtab, I took. Read the interest rates a bit better, I should have..."`
			`"On a Hulk, my first job in a spaceship was. Thought falling apart it was, I had, when to a Spindle it switched."`
			`"Liked my ship so much, a Kimek passenger did, that decided to live in it, they have. For 'transport' they still pay, so mind it I do not."`
			`"Jealous of you, I am. If a jump drive I had, bypass the gap between Saryd and Arach space, I could, and quickly earn a fortune, I would."`
			`"Agreed to transport some Saryd graduates for a 'field trip', I once did. Almost arrested, I was."`

news "arach civilian"
	location
		attributes "arach"
	name
		word
			"Local Arach"
	message
		word

> 			`"Arrested close to where my counsins work, the terrorists on the news were. Notice anything wrong with them, they never did."`

Reviewer: Typo fix.
```suggestion
			`"Arrested close to where my cousins work, the terrorists on the news were. Notice anything wrong with them, they never did."`
```

			`"Always rising, the sun in these broadcasts is, but a night shift I work, so see it for myself, I never can..."`
			`"To Ring of Wisdom, a researcher friend of mine has gone. Always there, our wisest mingle, but remain here, few do. Difficult for us, sometimes it is."`
			`"Always mixing up our Houses, Kimek and Saryd tourists are. Mind it, I do not, but angry, sometimes my friends get."`
			`"Learnt my language, some Kimek friends have, but so fast they speak, that difficult to understand them, it is."`

news "kimek civilian"
	location
		attributes "kimek"
	name
		word
			"Local Kimek"
	message
		word

> 			`"Arrested close to where my counsins work, the terrorists on the news were. Notice anything wrong with them, they never did."`

Reviewer: Same as above.
```suggestion
			`"Arrested close to where my cousins work, the terrorists on the news were. Notice anything wrong with them, they never did."`
```

			`"Always rising, the sun in these broadcasts is, but a night shift I work, so see it for myself, I never can..."`
			`"To Ring of Wisdom, a researcher friend of mine has gone. Always there, our wisest mingle, but remain here, few do. Difficult for us, sometimes it is."`
			`"Tired of dealing with faulty translation boxes, my sister got, so learning to speak Arachi, she is."`
			`"Much more reserved than us, the Saryds are. So long do they live for, that difficult to understand their habits, it can be."`

news "saryd civilian"
	location
		attributes "saryd"
	name
		word
			"Local Saryd"
	message
		word

> 			`"Arrested close to where my counsins work, the terrorists on the news were. Notice anything wrong with them, they never did."`

Reviewer: Same as above.
```suggestion
			`"Arrested close to where my cousins work, the terrorists on the news were. Notice anything wrong with them, they never did."`
```

	location
		government "Coalition" "Heliarch"
	name
		word
			"Heliarch agent"
	message
		word
			`"Protect and serve."`
			`"If any assistance you need, at your disposal I am."`
			`"If any trouble you find, report it to us you must."`
			`"Saryd, Kimek, Arach, human, no matter what species, to protect all of our citizens, committed we are."`
			`"Enjoy my post well enough, I do, but still to get a break during the Coalition games to watch them, I am."`
			`"Part of a Punisher crew, I am. Land, refuel, rest, and back to working on that magnificent ship, I am!"`
			`"Offered the position of elder, I actually have been. Only, not interested in that governing activity, I am."`
			`"Apologies, but make haste I must. Gone missing some garrison guns have, and back to cleaning Stalker engines I will be, if find them I don't!"`
			`"Difficult sometimes it is, to be a mediator. If careful with my words I am not, a Saryd might think an Arach talking about longcows has insulted their mother."`
			"An agent is trying to calm down a small crowd outside a local bar. After some minutes, the more agitated drunkards are convinced to accompany the agent to a nearby Heliarch office."

> 

Reviewer 2: ```suggestion
			"A pair of agents passes by you. Most passerby give them a wide berth."
			`"A pilot, I once was. Learned from an older sibling, I did."`

```

			`"Truly breathtaking, the sights of this world are. The cuisine, too, greatly enjoyed I have. If only the locals had a more civil sense of attire."`
			`"Climb up walls, the denizens of this world can. Otherworldly it is, to see them do so in the distinct architecture of some buildings here."`

news "coalition researcher"
	location
		government "Coalition"
		attributes "research"
	name
		word
			"Researcher"
	message
		word
			`"Pardon me, but providing me with a blood sample, would you mind? Interested in how the human immune system differs from our own, I am."`
			`"To Ring of Wisdom I frequently travel. Held there, many great conferences are."`
			`"Move to Ring of Wisdom, I wish to. So many advanced facilities for testing it has, so much easier to get a hold of complex chemicals..."`
			`"A Saryd Traveler I recently purchased. Not much of a pilot I am, but being a science vessel, adequate facilities the model still has. More importantly, no rude neighbors to interrupt my team's research, a spaceship has."`

> 

Reviewer 2: ```suggestion
			`A Saryd engrossed in a tablet displayed strange characters almost walks into you. "Excuse me," it mutters before continuing on its way.`
			`"A lecture in nanotechnology, I am here to present. Unclear on whether I would be teaching faculty or students, the invitation was."`
			`"Earned a degree in xenobiology, I did some time ago. Under a top researcher in the field, I work now."`

```

			`"Now that more information on yottrite we have, wonder if house Ablomab will ask for samples to experiment on, I do."`
			`"Kill for the papers on the yottrite research, I would, but still confidential the information is."`

news "coalition ringworld worker"
	location
		government "Heliarch"
		attributes "station"
	name
		word
			"Ringworld engineer"
			"Ringworld worker"
			"Construction supervisor"
			"Solar panel engineer"
			"Solar panel welder"
			"Maintenance worker"
	message
		word
			`"Just arrived here, I have. So glad that finally accepted for this job, I was!"`
			`"Food, lodging, an important job with a good pay, and a wonderful view. Ask for, what more could I?"`
			`"The means to finish construction of the two remaining rings, we still don't have, but one day, complete they shall be."`
			`"Wholly safe, our Coalition is, but here, even safer I feel. What is there to fear, when in an indestructible station you are?"`
			`"Still researching the alloys that make up the rings, our top scientists are, so only adding supplementary solar panels, we are."`

> 

Reviewer 2: ```suggestion
			"A small section of the area is roped off. An Arach has removed a floor panel and is busy reconfiguring circuitry below."
			`"Friend, please step aside." A Kimek gestures to a large cart filled with tools and equipment, and you move to let it pass.`

```

			`"Inspect every crate before it is put on a ship, the Heliarchs do. A tedious process, it is."`

news "coalition merchant"
	location
		government "Coalition" "Heliarch"
	name
		word
			"Saryd captain"
			"Kimek captain"
			"Arach captain"
			"Saryd pilot"
			"Kimek pilot"
			"Arach pilot"
	message
		word
			`"Enough money for my first ship, I didn't have, so split the cost of a Runabout, me and an online friend did. Picked another ship, we should have, because too difficult flying alone to their planet, it was."`
			`"Enough money for my first ship, I didn't have, so a loan from house Plumtab, I took. Read the interest rates a bit better, I should have..."`
			`"On a Hulk, my first job in a spaceship was. Thought falling apart it was, I had, when to a Spindle it switched."`
			`"Liked my ship so much, a Kimek passenger did, that decided to live in it, they have. For 'transport' they still pay, so mind it I do not."`
			`"Jealous of you, I am. If a jump drive I had, bypass the gap between Saryd and Arach space, I could, and quickly earn a fortune, I would."`
			`"Agreed to transport some Saryd graduates for a 'field trip', I once did. Almost arrested, I was."`

> 

Reviewer 2: ```suggestion
			"The spacefarer is assisting the rest of their crew in unloading cargo from a ship."
			`"Used to fly with me, my brother did, but now become a Heliarch, he has."`

```

	location
		government "Coalition" "Heliarch"
	name
		word
			"Heliarch agent"
	message
		word
			`"Protect and serve."`
			`"If any assistance you need, at your disposal I am."`
			`"If any trouble you find, report it to us you must."`
			`"Saryd, Kimek, Arach, human, no matter what species, to protect all of our citizens, committed we are."`
			`"Enjoy my post well enough, I do, but still to get a break during the Coalition games to watch them, I am."`
			`"Part of a Punisher crew, I am. Land, refuel, rest, and back to working on that magnificent ship, I am!"`
			`"Offered the position of elder, I actually have been. Only, not interested in that governing activity, I am."`
			`"Apologies, but make haste I must. Gone missing some garrison guns have, and back to cleaning Stalker engines I will be, if find them I don't!"`
			`"Difficult sometimes it is, to be a mediator. If careful with my words I am not, a Saryd might think an Arach talking about longcows has insulted their mother."`
			"An agent is trying to calm down a small crowd outside a local bar. After some minutes, the more agitated drunkards are convinced to accompany the agent to a nearby Heliarch office."

> 			"A pair of agents passes by you. Most passerby give them a wide berth."

Reviewer 2: ```suggestion
			"A pair of agents passes by you. Most of the spaceport crowd gives them a wide berth."
```
You mind if I change this? I was just rereading and didn't really like `passerby` following `passes by` so closely.

			`"Inspect every crate before it is put on a ship, the Heliarchs do. A tedious process, it is."`

news "coalition merchant"
	location
		government "Coalition" "Heliarch"
	name
		word
			"Saryd captain"
			"Kimek captain"
			"Arach captain"
			"Saryd pilot"
			"Kimek pilot"
			"Arach pilot"
	message
		word
			`"Enough money for my first ship, I didn't have, so split the cost of a Runabout, me and an online friend did. Picked another ship, we should have, because too difficult flying alone to their planet, it was."`
			`"Enough money for my first ship, I didn't have, so a loan from House Plumtab, I took. Read the interest rates a bit better, I should have..."`
			`"On a Hulk, my first job in a spaceship was. Thought falling apart it was, I had, when to a Spindle it switched."`
			`"Liked my ship so much, a Kimek passenger did, that decided to live in it, they have. For 'transport' they still pay, so mind it I do not."`
			`"Jealous of you, I am. If a jump drive I had, bypass the gap between Saryd and Arach space, I could, and quickly earn a fortune, I would."`

> 			`"Agreed to transport some Saryd graduates for a 'field trip', I once did. Almost arrested, I was."`

Reviewer 3: ```suggestion
			`"Agreed to transport some Saryd graduates for a 'field trip,' I once did. Almost arrested, I was."`
```
